logback.xml

```diff
@@ -0,0 +1,19 @@
+<configuration>
+    <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
```

Comment:
usually we using a short names like "thread->t"
You may found it in my presentation